### PR TITLE
Update status

### LINKED ISSUE #260: Update startpage
Update news/announcements.
Remove the "for updates join our discord" it takes to much space and the icon in the sidebar is enough.

### PR DESCRIPTION
**Pull request by matthias-luger**
fixes #260

### CODE REVIEW

File: src/components/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ function NavBar() {
                         <MenuItem icon={<ShareIcon />}><Link to="/ref" >Referral</Link></MenuItem>
                         <MenuItem icon={<PolicyIcon />}><Link to="/about" >Links / Legal</Link></MenuItem>
                         <MenuItem icon={<ChatIcon />}><Link to="/feedback" >Feedback</Link></MenuItem>
-                        <MenuItem icon={<HeadsetIcon />}><a href="https://discord.gg/Qm55WEkgu6"><div style={{ color: "#7289da" }}>Discord</div></a></MenuItem>
+                        <MenuItem icon={<img src="/discord_icon.svg" alt="" height="24px"></img>}><a href="https://discord.gg/Qm55WEkgu6"><div style={{ color: "#7289da" }}>Discord</div></a></MenuItem>
```
Comment (Ekwav) [inline src/components/NavBar/NavBar.tsx:131]: Are you sure this usage of the logo is compliant with the discord branding guidelines?

File: src/components/Startpage/Startpage.tsx [hunk 2]
```diff
@@ -182,68 +182,28 @@ function Startpage() {
...
                         <Card.Title><AnnouncementIcon /><span style={{ color: "#40ff00" }}> News / Announcements</span></Card.Title>
-                        <Card.Subtitle>Operations restored </Card.Subtitle>
+                        <Card.Subtitle>Hypixel API is offline</Card.Subtitle>
                     </Card.Header>
                     <Card.Body>
                         <p>
-                            We moved auctions to another server. 
-                            This should increase performance. 
-                            All systems are operational again.
+                            The Hypixel API is currently offline. So there wont be any new auctions or flips until its back up again.
```
Comment (Ekwav) [inline src/components/Startpage/Startpage.tsx:196]: Time estimated/where to check?

File: src/components/Startpage/Startpage.tsx [hunk 3]
```diff
@@ -182,68 +182,28 @@ function Startpage() {
...
                                 <ul>
-                                    <li className="changelog-item">The flipper auctions now have a small "?"-Symbol to see which auctions were used to calculate the shown median price. We would appreciate feedback to remove overpriced flips.</li>
-                                    <li className="changelog-item">Clear old flips for the filter. You can now clear all loaded flips for a better use of the scrollbar</li>
-                                    <li className="changelog-item">The new starting page</li>
-                                    <li className="changelog-item">Complete rework of the item price filter. You can now search for different metadata like Rarity or Pet Level depending on the item you have currently selected</li>
-                                    <li className="changelog-item">Style improvements</li>
+                                    <li className="changelog-item">New reworked sidebar</li>
+                                    <li className="changelog-item">Improved non-premium flipper (still delayed by a few minutes)</li>
```
Comment (Ekwav) [inline src/components/Startpage/Startpage.tsx:204]: Write this more posetively

**Review by Ekwav — CHANGES_REQUESTED** [review]
There are mergeconflicts

File: src/components/NavBar/NavBar.tsx (hunk 1 above)
Comment (matthias-luger) [reply src/components/NavBar/NavBar.tsx:131]: Its from the Discord Press Kit
https://discord.com/branding

File: src/components/Startpage/Startpage.tsx (hunk 3 above)
Comment (matthias-luger) [reply src/components/Startpage/Startpage.tsx:204]: no

**Review by Ekwav — APPROVED** [review]

**Pull request merged**